### PR TITLE
fix the pylint hang issue while performing sanity checks

### PR DESCRIPTION
**Pull request by kvignesh1420**
This PR addresses the issue: https://github.com/tensorflow/tensorflow/issues/41600 by updating the following:
- pylint version update for python3 while performing sanity checks resolves the issue where pylint hangs indefinitely during step 2 of the checklist in `tensorflow/tools/ci_build/ci_sanity.sh`.
- Provides a verbose message if pylint is not installed, so that we fail the step immediately.
-  Name the docker container in `tensorflow/tools/ci_build/ci_build.sh` for clarity purposes.

### CODE REVIEW

File: tensorflow/tools/ci_build/ci_sanity.sh [hunk 1]
```diff
@@ -126,6 +126,22 @@ do_pylint() {
 
   PYLINT_BIN="python3 -m pylint"
 
+  echo ""
+  echo "check whether pylint is available or not."
+  echo ""
+  ${PYLINT_BIN} --version
```
Comment (angerson) [inline tensorflow/tools/ci_build/ci_sanity.sh:132]: Small request: change [ to [[

**Review by angerson — CHANGES_REQUESTED** [review]
Thanks for the fix. I added one small request.

FWIW we don't use this system internally any more; we use tensorflow/tools/ci_build/release/ubuntu_16/sanity/build.sh (mirrored in tensorflow/tools/ci_build/rel/ubuntu/sanity.sh).

File: tensorflow/tools/ci_build/ci_sanity.sh (hunk 1 above)
Comment (kvignesh1420) [reply tensorflow/tools/ci_build/ci_sanity.sh:132]: addressed in the commit

**Comment by kvignesh1420** [comment]
@angerson since this instruction was present in the contributing docs to run the sanity checks using docker, I thought of fixing it. Just wondering if this should be changed or is it fine now that the fix is there. Let me know, thanks.

Ref: https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#running-sanity-check

**Comment by angerson** [comment]
Ah, thank you for the pointer. This will definitely help until we get a chance to review those docs.

**Review by angerson — APPROVED** [review]

**Pull request merged**